src/main/java/seedu/address/logic/commands/Command.java

```diff
@@ -17,6 +17,6 @@
      */
     public abstract CommandResult execute(ItemModel model) throws CommandException;
 
-    public abstract void reverse(ItemModel model) throws CommandException;
+    public abstract String getCommandWord();
```

Comment:
I think this should have been implemented in UndoableCommand but just change it in the next iteration.